test/Test/Sync.hs

```diff
@@ -0,0 +1,41 @@
+module Test.Sync
+       ( syncSpecs
+       ) where
+
+import IW.Core.Repo (RepoOwner (..), RepoName (..))
+import IW.Sync.Search (parseIssueUserData)
+import Test.Hspec (Spec, describe, it, shouldBe)
+
+import qualified GitHub  
+
+
+testGitHubIssueUrl :: GitHub.URL
+testGitHubIssueUrl = GitHub.URL "https://api.github.com/repos/owner123/repo123/issues/1"
+
+testBadGitHubIssueUrl1 :: GitHub.URL
+testBadGitHubIssueUrl1 = GitHub.URL "https://api.github.com/repos/owner123/issues/1"
```

Comment:
This URL looks exactly like `testGitHubIssueUrl` that's why tests are failing currently

Reply:
Well the `testBadGitHubIssueUrl1` is missing the `repoName` `repo123` in the URL.

Reply:
I''ll remove it for now.

Reply:
> I feel that Hakyll is more of a monolithic choice when it comes to static site generators

Ah, I see. Well, in this case there's nothing we can do, be need extremely strict URL parser to handle such cases. But, again, we expect GitHub to be fair.